capirca/lib/fortigatelocalin.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2022 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Fortigate local-in generator.
+
+This is a subclass of Fortigate generator.
+"""
+
+from capirca.lib import fortigate
+
+class FortigateLocalIn(fortigate.Fortigate):
+  """Fortigate local-in generator."""
+
+  _PLATFORM = 'fortigatelocalin'
+
+  def __str__(self):
+    fw_policies = self._get_fw_policies()
+
+    start_sys_settings = ['config sys setting']
+    start_addresses_v4 = ['config firewall address']
+    start_addresses_v6 = ['config firewall address6']
+    start_addrgrps_v4 = ['config firewall addrgrp']
+    start_addrgrps_v6 = ['config firewall addrgrp6']
+    start_services = ['config firewall service custom']
+    start_svcgrps = ['config firewall service group']
+    start_schedules = ['config firewall schedule onetime']
+    start_policies = ['config firewall local-in-policy']
```

Comment:
Just to confirm my understanding, is this the only difference between a Fortigate policy and a Fortigate local interface policy?

Are there any other platform differences?